ann_benchmarks/algorithms/pgvectorscale/config.yml

```diff
@@ -0,0 +1,33 @@
+float:
+  any:
```

Comment:
I don't believe `any` is applicable here, as the indexing only works for `cosine`.